repo/packages/H/history/0/marathon.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "history",
+  "cpus": 0.5,
+  "mem": 512,
+  "instances": 1,
+  "ports": [5055],
```

Comment:
The service itself uses whatever port is managed by mesos ($PORT). What is defined here is the service port (haproxy).